components/dashboard/src/settings/Account.tsx

```diff
@@ -47,7 +47,7 @@ export default function Account() {
 
         <PageWithSubMenu subMenu={settingsMenu}  title='Account' subtitle='Manage account and git configuration.'>
             <h3>Profile</h3>
-            <p className="text-base text-gray-500 pb-4 max-w-2xl">The following information will be used to set up git configuration. You can override git author name and email per project by using the default environment variables <CodeText>GIT_AUTHOR_NAME</CodeText> and <CodeText>GIT_COMMITTER_EMAIL</CodeText>.</p>
+            <p className="text-base text-gray-500 pb-4 max-w-2xl">The following information will be used to set up your workspace git configuration. If required, you can override these values using <a target="_blank" href="https://git-scm.com/book/en/v2/Git-Internals-Environment-Variables#_committing">environment variables</a>. </p>
```

Comment:
FYI, I believe that this was initially a typo.

When the workaround was described, it always mentioned both `GIT_AUTHOR_EMAIL` and `GIT_COMMITTER_EMAIL`: see https://github.com/gitpod-io/gitpod/issues/387

I believe when we wrote this sentence into the dashboard page, the developer accidentally wrote `GIT_AUTHOR_NAME` instead of `GIT_AUTHOR_EMAIL`.

I would personally much prefer it to just replace `GIT_AUTHOR_NAME` by `GIT_AUTHOR_EMAIL` (as initially intended), instead of being less explicit and just linking to the Git documentation (the URL might even go to 404 one day, leaving users with no information).

Reply:
in that case should i just change it to what i had before? or is there a reason we don't want to mention the `GIT_AUTHOR_NAME` and `GIT_COMMITTER_NAME` variables?

```suggestion
            <p className="text-base text-gray-500 pb-4 max-w-2xl">The following information will be used to set up git configuration. You can override git author name and email per project by using the default environment variables <CodeText>GIT_AUTHOR_NAME</CodeText>, <CodeText>GIT_COMMITTER_NAME</CodeText>, <CodeText>GIT_AUTHOR_EMAIL</CodeText> and <CodeText>GIT_COMMITTER_EMAIL</CodeText>.</p>
```

Reply:
I think they're just mentioned by accident, i.e. they are already set to a certain value by Gitpod, and I don't recall any time where the set values were a problem / needed to be customized by users (on that note, we'd also like users to be able to edit their name in https://gitpod.io/account).

However, the ability to customize the committer / author emails has come up very frequently, which is why we explicitly mention the `_EMAIL` variables in user accounts.

So, to me, mentioning also the `_NAME` variables is just a distraction (adds too much text / noise / potential confusion) and was never actually requested by users (I think -- I may be wrong though)

Reply:
On second thought, I don't have a very strong opinion on this. If you believe it is necessary to also mention the `_NAME` variables here, I can live with that. 😊

Reply:
In that case I think I'll add it back, because in my case I did have to edit both so I feel like it's useful to mention it